### DISCUSSION TITLE
Cabal Desktop 7.0.0

### DISCUSSION THREAD

**Discussion by nikolaiwarner** [cabal-club/cabal-desktop#322] [Features]
### Added
- Private messaging! Thanks @cblgh and @nikolaiwarner!

### Thanks for downloading Cabal Desktop! 🖤

<img width="646" alt="image" src="https://user-images.githubusercontent.com/40796/145597388-731babe8-3e7e-47f8-b2ed-592c5154e1c9.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/cabal-club/cabal-desktop/releases/tag/v7.0.0'>Cabal Desktop 7.0.0</a>.</em>

**Comment by todrobbins** [comment]
Did we lose the ability to change nicknames within the UI in a past version? I can't remember how to set my nick without `/nick` LOL

**Comment by drackord1** [comment]
se podrían enviar archivos ?

**Comment by angeleliseo** [comment]
If I want to install this on Mac which is the right artifact to pick?

**Comment by cblgh** [comment]
Which version of OSX? I think we have an undiagnosed issue with binaries for the new one

**Comment by todrobbins** [comment]
@angeleliseo I'm running 7.0.0 on macOS 13.4 without any issues:

https://github.com/cabal-club/cabal-desktop/releases/download/v7.0.0/cabal-desktop-7.0.0-mac.dmg
